assets/js/googlesitekit/notifications/register-defaults.js

```diff
@@ -569,6 +570,28 @@ export const DEFAULT_NOTIFICATIONS = {
 		},
 		isDismissible: true,
 	},
+	'module-recovery-alert': {
+		Component: ModuleRecoveryAlert,
+		priority: 320,
+		areaSlug: NOTIFICATION_AREAS.BANNERS_ABOVE_NAV,
+		viewContexts: [ VIEW_CONTEXT_MAIN_DASHBOARD ],
+		isDismissible: true,
+		checkRequirements: async ( { select, resolveSelect } ) => {
+			await resolveSelect( CORE_MODULES ).getRecoverableModules();
+			const recoverableModules =
+				select( CORE_MODULES ).getRecoverableModules();
```

Comment:
`resolveSelect` should be used directly for selectors with resolvers. This is the form we use where a selector does not have a resolver but relies on another selector that does, but in this case they'd reference different selectors.
```suggestion
			const recoverableModules = await resolveSelect( CORE_MODULES ).getRecoverableModules();
```